src/test/java/org/example/validator/ApplicationValidatorTest.java

```diff
@@ -0,0 +1,125 @@
+package org.example.validator;
+
+import org.example.exceptions.DoesNotExistException;
+import org.example.exceptions.InvalidException;
+import org.example.models.ApplicationRequest;
+import org.example.services.AuthService;
+import org.example.services.JobRoleService;
+import org.example.validators.ApplicationValidator;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+
+import java.sql.SQLException;
+
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.mockito.Mockito.doNothing;
+import static org.mockito.Mockito.doThrow;
+
+public class ApplicationValidatorTest {
+
+    AuthService authService = Mockito.mock(AuthService.class);
+    JobRoleService jobRoleService = Mockito.mock(JobRoleService.class);
+
+    ApplicationValidator applicationValidator =
+            new ApplicationValidator(authService, jobRoleService);
+
+
+    @Test
+    public void validateApplication_shouldNotThrowError_whenValidApplication() throws
+            SQLException, DoesNotExistException {
+        ApplicationRequest applicationRequest = new ApplicationRequest(
+                "user@kainos.com",
+                1,
+                "https://academy-job-portal-cvs.s3.eu-west-1.amazonaws.com/the_extra_semicolon/TestDocument.docx"
```

Comment:
:( All the animals are taking Kainos's jobs and the pirates are happy. Why have you made the pirates happy?